test/jdk/java/awt/List/MultiSelectionListHorizScrollbar.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 1999, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 4102881
+ * @summary Ensure multiple selection Lists have horizontal scrollbars when necessary
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual MultiSelectionListHorizScrollbar
+*/
+
+import java.awt.Frame;
+import java.awt.GridLayout;
+import java.awt.List;
+
+public class MultiSelectionListHorizScrollbar {
+
+    private static final String INSTRUCTIONS = """
+         Resize the frame so that the lists are not wide enough to fully display the lines of text they contain.
+         Once the lists are in this state, press pass if both lists display an horizontal scrollbar. Otherwise press fail.""";
```

Comment:
```suggestion
         Resize the frame so that the lists are not wide enough to fully display the lines of text they contain.
         Once the lists are in this state, press Pass if both lists display an horizontal scrollbar. Otherwise press Fail.""";
```